third_party/egeria/deploy.sh

```diff
@@ -3,7 +3,7 @@
 set -e
 set -x
 
-: ${WITHOUT_OPENSHIFT=false}
+: ${WITHOUT_OPENSHIFT=true}
```

Comment:
I believe it should be :=true and not =true

Reply:
Fixed. Thanks.

Reply:
If that's true then why not change the other variable too?

Reply:
fixed for the DONOT_DELETE_PERSISTENT_STORAGE flag also. I think = or := is equivalent. But it is better to have a consistent notation.